packages/preview/psl-thesis/0.1.0/template/main.typ

```diff
@@ -0,0 +1,126 @@
+#import "helpers.typ": *
+#import "@preview/psl-thesis:0.1.0": psl-thesis-covers
+#import "@preview/glossarium:0.5.1": make-glossary, register-glossary, print-glossary, gls, glspl
+#show: make-glossary
+
+#let glossary = (
+  (
+    key: "phd",
+    short: "PhD",
+    long: "philosophiæ doctor",
+  ),
+)
+
+#register-glossary(glossary)
+
+// Choose between fr and en to set the front cover language.
+#set text(lang: "fr", font: "Montserrat", size: 11pt)
+
+// Set chapter style for first level headings.
+#set heading(numbering: "1.1")
```

Comment:
I feel like a lot of the `show` and `set` rules in this file should be in the core of the package and not in the template itself. It only makes sense to have them here if you expect most users of your template to edit them. But this is up to you, and if you want to have them in the template it is possible.

Reply:
Indeed, putting them in the template and not in the package was intentional: I wanted the package to only provide the front and back covers of the thesis manuscript. This is similar to what the university provides: only the covers are standardized, but PhD students should be free to use any layout or style in their manuscript. Thus, I simply suggested an example style and layout for the template.

However, I'm not at all expert on how template packages are usually organized in Typst. Knowing that only covers are standardized at this university, what would you recommend?

Reply:
I think it makes sense to only have the covers in the package then.